addons/test_mail_full/tests/test_mail_performance.py

```diff
@@ -8,7 +8,7 @@
 from odoo.tools import mute_logger
 
 
-@tagged('mail_performance', 'post_install', '-at_install')
```

Comment:
why this change ?

Reply:
Good catch, sorry, it must have occurred during a rebase.